src/inspector/internal/compatibility/mu4inspectoradapter.h

```diff
@@ -60,15 +65,16 @@ class MU4InspectorAdapter : public IInspectorAdapter
     void showGridConfigurationDialog() override;
 
     // actions
-    void updatePageMarginsVisibility(const bool isVisible) override;
-    void updateFramesVisibility(const bool isVisible) override;
     void updateHorizontalGridSnapping(const bool isSnapped) override;
     void updateVerticalGridSnapping(const bool isSnapped) override;
-    void updateUnprintableElementsVisibility(const bool isVisible) override;
-    void updateInvisibleElementsDisplaying(const bool isVisible) override;
+    void updateInvisibleElementsDisplaying() override;
+    void updateUnprintableElementsVisibility() override;
+    void updateFramesVisibility() override;
+    void updatePageMarginsVisibility() override;
```

Comment:
How about renaming these to `toggle` instead of `update`?

Reply:
But they don't toggle anything, just send an action.